src/components/Layout/Layout.component.jsx

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
+import styled from 'styled-components';
+import Header from '../Header';
 
-import './Layout.styles.css';
+const Container = styled.div`
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+`;
 
-function Layout({ children }) {
-  return <main className="container">{children}</main>;
-}
+const Layout = ({ children }) => (
+  <Container>
+    <Header />
+    <main>{children}</main>
```

Comment:
You could create a <Body> component to avoid having to check for children prop.
Container component feels like something you'll be using along the project if not, you could try giving it a more specific name. If you are going to use it more than once, you can move it to a component folder and export it so it can be reused.